RFS/src/main/java/com/rfs/common/S3Repo.java

```diff
@@ -187,7 +187,7 @@ public void prepBlobFiles(ShardMetadata.Data shardMetadata) {
 
         String blobFilesS3Prefix = s3RepoUri.key + "indices/" + shardMetadata.getIndexId() + "/" + shardMetadata.getShardId() + "/";
 
-        logger.info("Downloading blob files from S3: s3://" + s3RepoUri.bucketName + "/" + blobFilesS3Prefix + " to " + shardDirPath);
+        logger.info("Downloading blob files from S3: s3://%s/%s to %s", s3RepoUri.bucketName, blobFilesS3Prefix, shardDirPath);
```

Comment:
I think we broadly are using `logger.atLevel(Info).setMessage(...).addArgument(() -> ...)....log();` @AndreKurait @gregschohn Did you have more context, is that a convention we should be strongly following?

Reply:
At some point Greg did a pass that changed all of the log messages to the fluent style `atLevel().setMessage()` and it meant we could run 15x faster or something absurd.

Reply:
ah, well, not quite 15x faster. but it did free up 50% of our CPU utilization.

https://github.com/opensearch-project/opensearch-migrations/pull/280

Reply:
I'm happy to have that be our convention going forward.  Will start swapping to that when I see the old style.